packages/plugin-calendar/test/integration/spec/calendar.js

```diff
@@ -0,0 +1,165 @@
+            mccoy.emailAddress,
+            spock.emailAddress
+          ],
+          mailboxInviteesInMeeting: [
+            {
+              id: mccoy.id,
+              invitee: mccoy.emailAddress,
+              type: `PERSON`,
+              responseType: `NO_RESPONSE`,
+              participantType: `OPTIONAL`
+            },
+            {
+              id: spock.id,
+              invitee: spock.emailAddress,
+              type: `PERSON`,
+              responseType: `UNKNOWN_RESPONSE`,
+              participantType: `REQUIRED`
+            }
+          ],
+          meetingIdentifier: {
+            id: `MeetingId-${meetingId}`
+          },
+          meetingTime: startInterval,
+          organizer: organizer.emailAddress,
+          startDate: startInterval,
+          subject: `test-plugin-calendar-meeting-${meetingId}`
+        },
+        meetingIdentifier: {
+          id: `MeetingId-${meetingId}`
+        }
+      };
+
+      return spark.request({
+        method: `POST`,
+        service: `whistler`,
+        resource: `calendarNotification`,
+        body: meetingParams,
+        qs: {
+          isinteg: false
+        }
+      })
+        .then((res) => {
+          createdMeeting = res.body;
+          console.log(`Created Meeting:\n `, JSON.stringify(createdMeeting, null, 4), `\n`);
+          assert.isDefined(createdMeeting);
+          assert.equal(createdMeeting.meeting.meetingIdentifier.id, meetingParams.meetingIdentifier.id);
+          assert.equal(createdMeeting.userUUID, meetingParams.userUUID);
+          assert.equal(createdMeeting.meeting.startDate, meetingParams.meeting.startDate);
+          assert.equal(createdMeeting.meeting.endDate, meetingParams.meeting.endDate);
+        });
+    });
+
+    beforeEach(`registers to wdm`, () => spark.device.register());
+
+    it(`retrieves the meeting list for default date range`, () => spark.calendar.get()
+      .then((meetings) => {
+        console.log(`Meetings fetched for default date range:\n `, JSON.stringify(meetings, null, 4), `\n`);
+        assert.equal(createdMeeting.meeting.icalUid, meetings[0].seriesId);
+        assert.equal(createdMeeting.userUUID, meetings[0].organizer);
+        assert.equal(createdMeeting.meeting.startDate, meetings[0].start);
+        assert.equal(moment(createdMeeting.meeting.endDate).diff(moment(createdMeeting.meeting.startDate), `minutes`), meetings[0].durationMinutes);
+
+        // Validate decryption of subject, location and agenda
+        assert.isDefined(meetings[0].encryptedSubject);
+        assert.equal(meetingParams.meeting.subject, meetings[0].encryptedSubject);
+        assert.isDefined(meetings[0].encryptedLocation);
+        assert.equal(meetingParams.meeting.location, meetings[0].encryptedLocation);
+      }));
+
+    // From: 1 week past, To : 3 weeks into the future
+    it(`retrieves the meeting list for specified date range`, () => spark.calendar.get({fromDate: fromInterval, toDate: toInterval})
```

Comment:
This test doesn't prove anything because it produces exactly the same result as the default values test.